### DISCUSSION TITLE
Is it possible to edit a document but not email after submit is pressed?

### DISCUSSION THREAD

**Discussion by ashwin9798** [docusealco/docuseal#206] [Q&A]
I love the interface. Just wanted to know if it's possible in the React component to get back the signed PDF as a blob in the onSubmit handler. This would be nice to implement custom workflows, e.g. I want to upload the signed PDF to my server instead of sending it as an email.

**Comment by omohokcoj** [comment]
@ashwin9798 signed documents can be retrieved via webhooks or API (documents are generated async and can't be passed in the onComplete callback).
https://www.docuseal.co/guides/download-signed-documents

Also you can use preview={true} react prop to show the signed form in preview mode.
